### PR TITLE
Blog: add pos release annoucement

### PR DESCRIPTION
**Pull request by RafaelGSS**
cc: @nodejs/releasers 

Refs: https://github.com/nodejs-private/node-private/issues/514

### CODE REVIEW

**Review by marco-ippolito — APPROVED** [review]

File: pages/en/blog/vulnerability/february-2024-security-releases.md [hunk 1]
```diff
@@ -1,12 +1,135 @@
...
+
+This security release includes the following dependency updates to address public vulnerabilities:
+
+- libuv 1.48.0 v21.x (CVE-2024-24806).
+- libuv on v18.x and v20.x has been patched to address CVE-2024-24806.
+- undici 5.28.3 on all release lines (CVE-2024-24758, GHSA-3787-6prv-h9w3).
+
+## OpenSSL Security updates
+
+This security release includes an OpenSSL to version 3.0.13+quic1 on all release lines to address advisories:
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/february-2024-security-releases.md]:
```suggestion
This security release includes an OpenSSL update to version 3.0.13+quic1 on all release lines to address advisories:
```

File: pages/en/blog/vulnerability/february-2024-security-releases.md [hunk 2]
```diff
@@ -1,12 +1,135 @@
...
+title: Wednesday February 14 2024 Security Releases
 layout: blog-post.hbs
-author: Rafael Gonzaga
+author: Rafael Gonzaga & Marco Ippolito
 ---
 
-## _(Update 14-February-2024)_ Security Release target Wednesday 14th
+## _(Update 14-February-2024)_ Security releases available
+
+Updates are now available for the v18.x, v20.x and 21.x Node.js release lines for the
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/february-2024-security-releases.md]:
```suggestion
Updates are now available for the v18.x, v20.x and v21.x Node.js release lines for the
```

File: pages/en/blog/vulnerability/february-2024-security-releases.md [hunk 3]
```diff
@@ -1,12 +1,135 @@
...
+
+The permission model protects itself against path traversal attacks by calling path.resolve() on any paths given by the user.
+If the path is to be treated as a Buffer, the implementation uses Buffer.from() to obtain a Buffer from the result of path.resolve().
+
+By monkey-patching Buffer internals, namely, Buffer.prototype.utf8Write, the application can modify the result of path.resolve(), which leads to a path traversal vulnerability.
+
+Impacts:
+
+- This vulnerability affects all users using the experimental permission model in active release
+  lines: 20.x and, 21.x.
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/february-2024-security-releases.md]:
```suggestion
  lines: 20.x and 21.x.
```

File: pages/en/blog/vulnerability/february-2024-security-releases.md [hunk 4]
```diff
@@ -1,12 +1,135 @@
...
+
+On Linux, Node.js ignores certain environment variables if those may have been set by an unprivileged user while the process is running with elevated privileges with the only exception of CAP_NET_BIND_SERVICE.
+
+Due to a bug in the implementation of this exception, Node.js incorrectly applies this exception even when certain other capabilities have been set.
+
+This allows unprivileged users to inject code that inherits the process's elevated privileges.
+
+Impacts:
+
+- This vulnerability affects all users in all active release lines: 18.x, 20.x and, 21.x.
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/february-2024-security-releases.md]:
```suggestion
- This vulnerability affects all users in all active release lines: 18.x, 20.x, and 21.x.
```

File: pages/en/blog/vulnerability/february-2024-security-releases.md [hunk 5]
```diff
@@ -1,12 +1,135 @@
...
+A vulnerability in Node.js HTTP servers allows an attacker to send a specially crafted HTTP request with chunked encoding,
+leading to resource exhaustion and denial of service (DoS).
+
+The server reads an unbounded number of bytes from a single connection, exploiting the lack of limitations on chunk extension bytes.
+
+The issue can cause CPU and network bandwidth exhaustion, bypassing standard safeguards like timeouts and body size limits.
+
+Impacts:
+
+- This vulnerability affects all users in all active release lines: 18.x, 20.x and, 21.x.
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/february-2024-security-releases.md]:
```suggestion
- This vulnerability affects all users in all active release lines: 18.x, 20.x, and 21.x.
```

File: pages/en/blog/vulnerability/february-2024-security-releases.md [hunk 6]
```diff
@@ -1,12 +1,135 @@
...
+
+## setuid() does not drop all privileges due to io_uring (CVE-2024-22017) - (High)
+
+setuid() does not affect libuv's internal io_uring operations if initialized before the call to setuid().
+
+This allows the process to perform privileged operations despite presumably having dropped such privileges through a call to setuid().
+
+Impacts:
+
+- This vulnerability affects all users in all active release lines: 18.x, 20.x and, 21.x.
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/february-2024-security-releases.md]:
```suggestion
- This vulnerability affects all users in all active release lines: 18.x, 20.x, and 21.x.
```

File: pages/en/blog/vulnerability/february-2024-security-releases.md [hunk 7]
```diff
@@ -1,12 +1,135 @@
...
+
+A vulnerability in the privateDecrypt() API of the crypto library, allowed a covert timing side-channel during PKCS#1 v1.5 padding error handling.
+
+The vulnerability revealed significant timing differences in decryption for valid and invalid ciphertexts.
+
+This poses a serious threat as attackers could remotely exploit the vulnerability to decrypt captured RSA ciphertexts or forge signatures, especially in scenarios involving API endpoints processing Json Web Encryption messages.
+
+Impacts:
+
+- This vulnerability affects all users in all active release lines: 18.x, 20.x and, 21.x.
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/february-2024-security-releases.md]:
```suggestion
- This vulnerability affects all users in all active release lines: 18.x, 20.x, and 21.x.
```

File: pages/en/blog/vulnerability/february-2024-security-releases.md [hunk 8]
```diff
@@ -1,12 +1,135 @@
...
+for fixing it.
+
+## Multiple permission model bypasses due to improper path traversal sequence sanitization (CVE-2024-21891) - (Medium)
+
+Node.js depends on multiple built-in utility functions to normalize paths provided to node:fs functions, which can be overwitten with user-defined implementations leading to filesystem permission model bypass through path traversal attack.
+
+Impacts:
+
+- This vulnerability affects all users using the experimental permission model in active release
+  lines: 20.x and, 21.x.
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/february-2024-security-releases.md]:
```suggestion
  lines: 20.x and 21.x.
```

File: pages/en/blog/vulnerability/february-2024-security-releases.md [hunk 9]
```diff
@@ -1,12 +1,135 @@
...
+## Improper handling of wildcards in --allow-fs-read and --allow-fs-write (CVE-2024-21890) - (Medium)
+
+The Node.js Permission Model does not clarify in the documentation that wildcards should be only used as the last character of a file path.
+
+For example: `--allow-fs-read=/home/node/.ssh/*.pub` will ignore pub and give access to everything after .ssh/.
+
+Impacts:
+
+- This misleading documentation affects all users using the experimental permission model in active release
+  lines: 20.x and, 21.x.
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/february-2024-security-releases.md]:
```suggestion
  lines: 20.x and 21.x.
```

**Review by MattIPv4 — APPROVED** [review]

**Review by UlisesGascon — APPROVED** [review]
LGTM!

File: pages/en/blog/vulnerability/february-2024-security-releases.md [hunk 10]
```diff
@@ -1,12 +1,135 @@
...
+
+## setuid() does not drop all privileges due to io_uring (CVE-2024-22017) - (High)
+
+setuid() does not affect libuv's internal io_uring operations if initialized before the call to setuid().
+
+This allows the process to perform privileged operations despite presumably having dropped such privileges through a call to setuid().
+
+Impacts:
+
+- This vulnerability affects all users in all active release lines: 18.x, 20.x, and 21.x.
```
Comment (richardlau) [inline pages/en/blog/vulnerability/february-2024-security-releases.md]:
```suggestion
- This vulnerability affects all users in all active release lines: 20.x, and 21.x.
```

Node.js 18 contains libuv 1.44.2 which doesn't support io_uring.

File: pages/en/blog/vulnerability/february-2024-security-releases.md [hunk 11]
```diff
@@ -1,12 +1,135 @@
...
+
+## setuid() does not drop all privileges due to io_uring (CVE-2024-22017) - (High)
+
+setuid() does not affect libuv's internal io_uring operations if initialized before the call to setuid().
+
+This allows the process to perform privileged operations despite presumably having dropped such privileges through a call to setuid().
+
+Impacts:
+
+- This vulnerability affects all users in all active release lines: 20.x, and 21.x.
```
Comment (richardlau) [inline pages/en/blog/vulnerability/february-2024-security-releases.md]:
```suggestion
- This vulnerability affects all users in active release lines: 20.x, and 21.x.
```

**Review by marco-ippolito — APPROVED** [review]

**Review by AugustinMauroy — APPROVED** [review]

**Pull request merged**